Apps/Sandcastle/gallery/development/Shadows.html

```diff
@@ -306,12 +306,31 @@
     updateLightDirection();
     updateShadows();
 }
+
+// TODO : this is a very hacky way to reset the globe shaders when shadow properties change
```

Comment:
This is just for the Sandcastle example, right?  If a user needs to do this in practice, we can come up with something cleaner.